torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3293,6 +3301,15 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                    dtypesIfCUDA=None,
                    dtypesIfROCM=None,
                    supports_autograd=False),
+    OpInfo('cdist',
+           dtypes=floating_types(),
+           supports_out=False,
+           check_gradgrad=False,
+           skips=(
+               # jit consistency always checks gradgrad
```

Comment:
Can we add the check to the test along with a `# TODO: ...` comment explaining that it'd be nice if the test could avoid gradgrad checks if the op doesn't support gradgrad?

Reply:
I wanted to update it, but it happens here: https://github.com/pytorch/pytorch/blob/5cab3b9cf628ba44b599fa7da4c06404986dd656/torch/testing/_internal/common_jit.py#L40
Which is jit specific code. And my quick attempt to modify it lead to more things blowing up in other tests.

But if you think that we should add it there, happy to do it. Just let me know where it would be most appropriate.

Reply:
I think what we can do for is update this logic:

https://github.com/pytorch/pytorch/blob/9be2cabc4575f86e53c2b66ea277084f5c003c0a/test/test_ops.py#L346

so that requires grad is only true if both grad and grad grad are supported, and add the comment there explaining why this is imperfect (because we could still test grad and not grad grad if the jit helper was updated to understand that nuance)

Reply:
That what I tried, this is what it leads to:
```
torch._C._jit_check_alias_annotation(CU.the_method.graph, tuple(tensors), aten_name)
RuntimeError: aliasOp != torch::jit::getOperatorAliasMap().end()INTERNAL ASSERT FAILED at "../torch/csrc/jit/passes/utils/check_alias_annotation.cpp":155, please report a bug to PyTorch.
```

That's why I just disabled it :D

Reply:
Huh, that's bizarre that setting _requires_grad to False here results in an aliasing error. 

In that case, would you just skip in the test:

```
# TODO: fix this
if _requires_grad and not op.supports_gradgrad:
  self.skipTest("skipped! Test does not handle ops that don't support gradgrad properly!")
```

?